Vesper-Agents-Suite/unhealthy-borrow-reporter/src/agent.ts

```diff
@@ -12,9 +12,9 @@ import {
 import { createFinding, contains } from "./utils";
 import BigNumber from "bignumber.js";
 const STRATEGY_KEY = "_Strategies"
-const COMPOUND_STRATEGIES = ["BenqiXYStrategy", "TraderJoeXYStrategy", "CompoundXYStrategy", "IronBankXYStrategy"]
-const AAVE_STRATEGIES = ["AaveXYStrategy"]
-const MAKER_STRATEGIES = ["MakerStrategy", "Maker-Strategy"]
+const COMPOUND_STRATEGIES = ["BenqiXY", "TraderJoeXY", "CompoundXY", "IronBankXY", "Compound_Vesper_Xy", "TraderJoe_Vesper_Xy", "Benqi_Vesper_Xy"]
```

Comment:
we still have old strategy in production, so we may want to keep old names too.

Reply:
@rokso  - It will work for old strategies as well. The listed string here is a substring of the actual strategy names.